README.md

```diff
@@ -7,38 +7,54 @@ Develop: [![Build Status](https://travis-ci.org/sansible/newrelic_agent_java.svg
 * [Tags](#tags)
 * [Examples](#examples)
 
-This role ...
+This role installs Newrelic Java Agent.
+
+For information how to enable newrelic.jar please check
+https://docs.newrelic.com/docs/agents/java-agent/installation/include-java-agent-jvm-argument
+
+
 
 
 ## Installation and Dependencies
 
-To install run `ansible-galaxy install sansible.newrelic_agent_java` or add this to your
-`roles.yml`.
+To install run `ansible-galaxy install sansible.newrelic_agent_java` or add this to
+your `roles.yml`.
 
 ```YAML
 - name: sansible.newrelic_agent_java
-  version: v1.0
+  version: v1.0.0-latest
```

Comment:
What does the ```-latest``` but do?

Reply:
an approach to make it semver naming compliant